apps/console/package.json

```diff
@@ -23,7 +23,13 @@
         "start": "npx run-script-os",
         "start:win32": "..\\..\\node_modules\\.bin\\webpack serve --mode development --inline --hot --open --env NODE_ENV=development",
         "start:default": "../../node_modules/.bin/webpack serve --mode development --inline --hot --open --env NODE_ENV=development",
-        "test": "npx jest --passWithNoTests"
+        "test": "npx jest --passWithNoTests",
+        "compileI18n": "npx run-script-os",
```

Comment:
Shall we say `compile:i18n` & `clean:i18n:dist` instead. WDYT?

Reply:
Resolved by a85abe0